src/OpenTK.GLControl/GLControl.cs

```diff
@@ -181,6 +196,12 @@ protected override CreateParams CreateParams
         /// <param name="e">Not used.</param>
         protected override void OnHandleCreated(EventArgs e)
         {
+            if (FailedCreateContext)
```

Comment:
Is this check needed? If OnHandleCreated is raised again shouldn't we try to create the context again with the new handel?

Reply:
If you fail once I would not like to try this again. Reasoning: Why would it going to work later, if you initialization logic is correctly sorted?